pages/deployment/[uuid].tsx

```diff
@@ -49,10 +49,16 @@ function CodeLine({ line }) {
         style={{
           cursor: 'pointer',
         }}
+        role='button'
+        tabIndex={index}
```

Comment:
Tab index should basically never be greater than zero.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

> Warning: Avoid using tabindex values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.

Reply:
Resolved